backend/models/Locations.ts

```diff
@@ -40,6 +40,9 @@ const schema = new mongoose.Schema({
   author: { type: String, required: true },
   createdAt: { type: Date },
   updatedAt: { type: Date }
+},

```

Comment:
Is it necessary to keep these two fields in the scheme declaration, in case you've already declared the timestamp?

Reply:
no it is not necessary, i removed it. i also removed it from the interface, and  it works